### DISCUSSION TITLE
Bluetooth headset buttons control bug when more than one media is playing.

### DISCUSSION THREAD

**Discussion by sukirti2004** [blueman-project/blueman#2454] [Q&A]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

My Realme Buds wireless 3 has 3 buttons Vol+, Vol-, and a multifunction button(play, pause, next track, previous track).
Everything works except next and prev track which works with by pressing multifunction  button  2 times and 3 times respectively.
Even My Sony headphones next and prev button combination doesn't work.

Also there is a bug in play/pause, whenever there is more than one media playing It doesn't change between the media playing after the previous media has been paused and minimized.

Update:  After using it for a while the buttons do work properly when only single media is playing but somehow it isn't able to change the media controls  to latest media playing and sticks to the previous media only.

**Comment by cschramm** [comment]
Nothing blueman would be involved in. You can check registered key presses e.g. with `xev`, configure mappings for your desktop and eventually some component has to catch those controls.

**Reply by sukirti2004** [reply]
After using it for a while I have realised that buttons do work but the problem of not being able to detect the latest media playing is still an issue.  Even after changing the launcher player manually it still somehow sticks to the previous media playing.
